mathics/eval/hyperbolic.py

```diff
@@ -3,8 +3,13 @@
 """
 from mathics.core.convert.sympy import from_sympy
 
+from sympy import Symbol
```

Comment:
Maybe it would be better to use an alias `from sympy import Symbol as SympySymbol` for clarity.